spec/features/acceptance/profile_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+include Warden::Test::Helpers
+
+describe 'profile', type: feature do
+  #self.use_transactional_fixtures = false #only necessary when using selenium with js:true
+  before :each do
+    @current_user = FactoryGirl.create(:user)
+    login_as(@current_user, scope: :user)
+  end
+
+  it "shows only given details" do
+    visit "/users/#{@current_user.id}/"
+    within('body/div.container') do
+      expect(page).to have_content 'First Name'
```

Comment:
Der Lehrstuhl wurde zu 'Research Group' umbenannt und wird in Zeile 80, 81 getestet